modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -150,20 +151,72 @@
     /** */
     private File customDiagnosticDir;
 
+    /** */
+    private static final int serverNodeCnt = 2;
+    /** */
+    private static final int clientNodeCnt = 1;
+
+    /** */
+    protected static IgniteEx crd;
+    /** */
+    private static IgniteEx client;
+
+    /** {@inheritDoc} */
+    @Override protected void beforeTestsStarted() throws Exception {
+        crd = startGrids(serverNodeCnt);
+        client = startGrid("client");
```

Comment:
Create constant in abstract class for client names prefix.

Reply:
Why?